adam/experiment/log_experiment.py

```diff
@@ -210,6 +215,10 @@ def learner_factory_from_params(
                 beam_size=beam_size,
                 language_mode=language_mode,
             ),
+            number_learner=QuantifierTemplateLearner.pretrained_for_language_mode(
```

Comment:
I haven't tested at all that this doesn't cause problems for other curricula, which is the main reason this hasn't been merged to master yet. It could be totally buggy.